projects/plugins/jetpack/extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -56,14 +65,16 @@ function load_assets( $attr, $content ) {
 		? get_current_blog_id()
 		: Jetpack_Options::get_option( 'id' );
 	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
-	$classes         = Blocks::classes( FEATURE_NAME, $attr );
```

Comment:
I take that back. As described in https://github.com/Automattic/jetpack/pull/22622/files#r802459668 the `apply_block_supports()` function now handles everything.

Reply:
This is good to know. It may be a good follow-up janitorial PR to update our `classes` method to use `apply_block_supports()`?

Reply:
> This is good to know. It may be a good follow-up janitorial PR to update our classes method to use apply_block_supports()?

Yes, that can be done.